pytorch_lightning/strategies/tpu_spawn.py

```diff
@@ -68,22 +68,8 @@ def __init__(
             precision_plugin=precision_plugin,
         )
         self.debug = debug
-        self.tpu_local_core_rank = 0
-        self.tpu_global_core_rank = 0
         self.start_method = "fork"
 
-    @property
-    def global_rank(self) -> int:
-        return self.tpu_global_core_rank
-
-    @property
-    def local_rank(self) -> int:
-        return self.tpu_local_core_rank
-
-    @property
-    def world_size(self) -> int:
```

Comment:
Why do you remove world_size ?

Reply:
Because world size is set via `set_world_ranks` and then handled by LightningClusterEnvironment. I didn't remove it completely, it's just handled by the super class

Reply:
Yes, but are you sure `self.node_rank * self.num_processes` are the right values ?